app/connectors/DesConnector.scala

```diff
@@ -37,25 +39,22 @@ class DesConnector @Inject() (config: AppConfig, httpClient: HttpClient) (implic
 
   def subscribe(lisaManager: String, payload: JsValue)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
     val uri = s"$subscriptionUrl/$lisaManager/subscription"
-    logger.info(s"DES Connector post subscription $uri")
-    val headerCarrier = addCorrelationId(hc)
-    httpClient.POST(uri, payload, desHeaders)(implicitly, httpReads, headerCarrier, implicitly) map { res => res } recover {
-      case e: Exception =>
-        logger.error(s"Error in DesConnector subscribe: ${e.getMessage}")
-        throw e
-    }
+    httpPost(uri, payload, "subscribe", "subscription")
   }
 
   def register(utr: String, payload: JsValue)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
     val uri = s"$registrationUrl/utr/$utr"
-    logger.info(s"DES Connector post registerOnce $uri")
+    httpPost(uri, payload, "register", "registerOnce")
+  }
+
+  private def httpPost(uri: String, payload: JsValue, urlType: String, connectorLog: String)(implicit hc: HeaderCarrier) = {
```

Comment:
For some reason the original logging message preferred **registerOnce** rather than **register** similarly **subscription** rather than **subscribe**

We can happily move away from that to ensure the logging is consistent with the end of the **uri** path

Reply:
I think we can leave it, it seems fine